Cargo.lock

```diff
@@ -337,14 +337,14 @@ checksum = "64b728d511962dda67c1bc7ea7c03736ec275ed2cf4c35d9585298ac9ccf3b73"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn 2.0.104",
+ "syn 2.0.106",
 ]
 
 [[package]]
 name = "alloy-rpc-types-any"
-version = "1.0.13"
+version = "1.0.19"
```

Comment:
I see alloy is updated, could you check that connector is aware of this? from what I remember they had to use the same alloy package as us